GIT improperly converts end-of-line characters in new reference file used in inject codegen tests

## Environment Details
* Helidon Version: current main branch (4.2)
* Helidon SE 
* JDK version: 21.0.4
* OS: Windows 11

----------

## Problem Description
Tests for Helidon recently added in PR 9642 added reference files: 

- integrations\oci\sdk\codegen\src\test\resources\expected\Objectstorage__Oci_Client._java_
- integrations\oci\sdk\codegen\src\test\resources\expected\Objectstorage__Oci_ClientBuilder._java_

These files are compared byte for byte in the module:  Helidon Integrations OCI Injection Codegen.
GIT pulls these files to Windows, it applies the rule to change the end-of-line character for Windows since they are text files. This causes the unit test to fail, since the new line character differs for Windows.

## Steps to reproduce
[//]: # Git Pull main branch, Windows platform
[//]: # Build (mvn clean install)

```
...
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.helidon.integrations.oci.sdk.codegen.OciInjectionProcessorObserverTest
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.922 s <<< FAILURE! - in io.helidon.integrations.oci.sdk.codegen.OciInjectionProcessorObserverTest
[ERROR] io.helidon.integrations.oci.sdk.codegen.OciInjectionProcessorObserverTest.generatedInjectionArtifactsForTypicalOciServices  Time elapsed: 0.499 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: "/*\r\n * Copyright (c) 2025 Oracle and/or its affiliates.\r\n *\r\n * Licensed under the Apache License, Version 2.0 (the \"License\");\r\n * you may not use this file except in compliance with the License.\r\n * You may obtain a copy of the License at\r\n *\r\n *     http://www.apache.org/licenses/LICENSE-2.0\r\n *\r\n * Unless required by applicable law or agreed to in writing, software\r\n * distributed under the License is distributed on an \"AS IS\" BASIS,\r\n * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\r\n * See the License for the specific language governing permissions and\r\n * limitations under the License.\r\n */\r\n\r\npackage io.helidon.integrations.generated.com.oracle.bmc.objectstorage;\r\n\r\nimport java.util.Optional;\r\n\r\nimport io.helidon.common.Generated;\r\nimport io.helidon.common.Weight;\r\nimport io.helidon.service.registry.Lookup;\r\nimport io.helidon.service.registry.Service;\r\n\r\nimport com.oracle.bmc.auth.AbstractAuthenticationDetailsProvider;\r\nimport com.oracle.bmc.objectstorage.ObjectStorage;\r\nimport com.oracle.bmc.objectstorage.ObjectStorageClient;\r\n\r\n@Generated(value = \"io.helidon.integrations.oci.sdk.codegen.OciInjectionCodegenObserver\", trigger = \"com.oracle.bmc.objectstorage.ObjectStorage\")\r\n@Service.Singleton\r\n@Weight(99.0D)\r\n@Service.ExternalContracts(ObjectStorage.class)\r\nclass ObjectStorage__Oci_Client implements Service.InjectionPointFactory<ObjectStorageClient> {\r\n\r\n    private final Service.InjectionPointFactory<AbstractAuthenticationDetailsProvider> authProvider;\r\n    private final Service.InjectionPointFactory<ObjectStorageClient.Builder> builderProvider;\r\n\r\n    @Service.Inject\r\n    @Deprecated\r\n    ObjectStorage__Oci_Client(Service.InjectionPointFactory<AbstractAuthenticationDetailsProvider> authProvider, Service.InjectionPointFactory<ObjectStorageClient.Builder> builderProvider) {\r\n        this.authProvider = authProvider;\r\n        this.builderProvider = builderProvider;\r\n    }\r\n\r\n    @Override\r\n    public Optional<Service.QualifiedInstance<ObjectStorageClient>> first(Lookup query) {\r\n        return Optional.of(Service.QualifiedInstance.create(builderProvider.first(query).orElseThrow().get().build(authProvider.first(query).orElseThrow().get())));\r\n    }\r\n\r\n}"
     but: was "/*\n * Copyright (c) 2025 Oracle and/or its affiliates.\n *\n * Licensed under the Apache License, Version 2.0 (the \"License\");\n * you may not use this file except in compliance with the License.\n * You may obtain a copy of the License at\n *\n *     http://www.apache.org/licenses/LICENSE-2.0\n *\n * Unless required by applicable law or agreed to in writing, software\n * distributed under the License is distributed on an \"AS IS\" BASIS,\n * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n * See the License for the specific language governing permissions and\n * limitations under the License.\n */\n\npackage io.helidon.integrations.generated.com.oracle.bmc.objectstorage;\n\nimport java.util.Optional;\n\nimport io.helidon.common.Generated;\nimport io.helidon.common.Weight;\nimport io.helidon.service.registry.Lookup;\nimport io.helidon.service.registry.Service;\n\nimport com.oracle.bmc.auth.AbstractAuthenticationDetailsProvider;\nimport com.oracle.bmc.objectstorage.ObjectStorage;\nimport com.oracle.bmc.objectstorage.ObjectStorageClient;\n\n@Generated(value = \"io.helidon.integrations.oci.sdk.codegen.OciInjectionCodegenObserver\", trigger = \"com.oracle.bmc.objectstorage.ObjectStorage\")\n@Service.Singleton\n@Weight(99.0D)\n@Service.ExternalContracts(ObjectStorage.class)\nclass ObjectStorage__Oci_Client implements Service.InjectionPointFactory<ObjectStorageClient> {\n\n    private final Service.InjectionPointFactory<AbstractAuthenticationDetailsProvider> authProvider;\n    private final Service.InjectionPointFactory<ObjectStorageClient.Builder> builderProvider;\n\n    @Service.Inject\n    @Deprecated\n    ObjectStorage__Oci_Client(Service.InjectionPointFactory<AbstractAuthenticationDetailsProvider> authProvider, Service.InjectionPointFactory<ObjectStorageClient.Builder> builderProvider) {\n        this.authProvider = authProvider;\n        this.builderProvider = builderProvider;\n    }\n\n    @Override\n    public Optional<Service.QualifiedInstance<ObjectStorageClient>> first(Lookup query) {\n        return Optional.of(Service.QualifiedInstance.create(builderProvider.first(query).orElseThrow().get().build(authProvider.first(query).orElseThrow().get())));\n    }\n\n}"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at io.helidon.integrations.oci.sdk.codegen.OciInjectionProcessorObserverTest.generatedInjectionArtifactsForTypicalOciServices(OciInjectionProcessorObserverTest.java:82)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.apache.maven.surefire.junitplatform.LazyLauncher.execute(LazyLauncher.java:50)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:184)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:148)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:122)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)


```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2022 Oracle and/or its affiliates.
+# Copyright (c) 2022, 2025 Oracle and/or its affiliates.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.

--- a/.gitattributes
+++ b/.gitattributes
@@ -15,3 +15,4 @@
 
 # don't let GIT perform end of line mapping on these files
 enum-test*.txt -text
+Object*._java_ -text

